src/components/common/FileUploadComponent.tsx

```diff
@@ -429,7 +432,9 @@ export function FileUploadComponent(props: {
   }
 
   const amountFilesInfo =
-    maxFiles > 1 ? <span>Max: {maxFiles} file(s)</span> : null;
+    maxFiles > 1 ? (
+      <span className={classes.nbrFilesText}>Max: {maxFiles} file(s)</span>
```

Comment:
As I reviewed recently some code from @jekabs-karklins and we decided that probably it is a bit better and cleaner to use more the `Box` component that comes from MUI. So this will be much more readable if it is written like: `<Box component="span" display="block">Max: {maxFiles} file(s)</Box>` and remove the custom class that we add in the makeStyles block. This way I think we keep makeStyles smaller and the code is more readable when you see it you don't have to scroll up to see the style definitions.